### DISCUSSION TITLE
Standard Tailwind Colors

### DISCUSSION THREAD

**Discussion by LeonWehrhahn** [saadeghi/daisyui#857] [Q&A] [answered]
Is it a normal behaviour that standard tailwind colors don't work? When I style a div with **bg-green-300** it is not applying the color. Only daisy ui colors are working (**bg-secondary**).

**Comment by saadeghi** [comment, marked as answer]
No. All colors should work. daisyUI color names are just additional color names and they don't disable other colors.

Can you share your `tailwind.config.js` file?

**Reply by LeonWehrhahn** [reply]
Hi, thanks for the reply. I have reconfigured everything. Now standard tailwind colors are working again.

**Reply by artslimedev** [reply]
how did you configure this to work properly? I can't seem to figure it out.
